includes/reader-revenue/class-reader-revenue-emails.php

```diff
@@ -120,6 +121,35 @@ public static function add_email_configs( $configs ) {
 				]
 			),
 		];
+		$configs[ self::EMAIL_TYPES['WELCOME'] ]      = [
+			'name'                   => self::EMAIL_TYPES['WELCOME'],
+			'label'                  => __( 'Welcome', 'newspack-plugin' ),
+			'description'            => __( "Email sent to the donor after they've registered while donating.", 'newspack-plugin' ),
```

Comment:
In my testing, this doesn't only apply to donations, but to any transaction (this is correct according to the scope of the feature request task). 

```suggestion
			'description'            => __( "Email sent to new reader accounts registered during a transaction.", 'newspack-plugin' ),
```

Reply:
Good suggestion! Updated in https://github.com/Automattic/newspack-plugin/pull/3150/commits/c58f6399821ce0dcd1262c47d8d609df225e4a42